validator/validation/readers.py

```diff
@@ -27,10 +26,45 @@
 
 __logger = logging.getLogger(__name__)
 
+
+class SMAPL3_AM_PM:
```

Comment:
This class should inherit from pynetcf GriddedNcIndexedRaggedTs class

Reply:
actually, I don't understand why this wrapper is necessary. Just use SMAPL3_V9Reader?